src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -29,6 +31,26 @@ spec:
         remoteServiceAccount: pepr-uds-core
         port: 8080
 
+      - description: "Waypoint health"
```

Comment:
Note to reviewers: Without this, Prometheus fails to scrape metrics from Waypoint and it shows this target as failed.